### PR TITLE
[Meta] Update README with new contribution guide

### PR DESCRIPTION
**Pull request by randomicecube**

### CODE REVIEW

File: README.md
Comment (diogotcorreia) [inline README.md:38]: I don't think this is needed, only contributors have write access, therefore HTTPS is always enough for everyone else

File: README.md
Comment (diogotcorreia) [inline README.md:50]: This is not true, frontmatter metadata must be present (`path`, and optionally `type`)

File: README.md
Comment (diogotcorreia) [inline README.md:61]: Maybe it'd be nice to put here a full list of available types (or maybe just in the in-depth docs in the wiki)

File: README.md
Comment (diogotcorreia) [inline README.md:74]: This is not necessary anymore, `yarn format` is an alias to that

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
Also, it's probably a good idea to link the wiki here, since it's a more complete guide

File: README.md
Comment (randomicecube) [reply README.md:74]: TIL, ty!

File: README.md
Comment (randomicecube) [reply README.md:61]: I'd argue it'd be fine for it to be fully documented in the Wiki (and for us to include only the essentials here), but it's your call

**Comment by randomicecube** [comment]
> Also, it's probably a good idea to link the wiki here, since it's a more complete guide

This is now referenced at the end of the file (as a sort of foot note), tell me if you think it should be higher up

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**
